ChangeLog

```diff
@@ -144,6 +144,8 @@ Bugfix
      operations psa_mac_compute() and psa_mac_sign_setup().
    * The key usage flags PSA_KEY_USAGE_VERIFY_MESSAGE now allows the MAC
      operations psa_mac_verify() and psa_mac_verify_setup().
+   * Fix a bug that ECP multiplication is wrongly rejecting valid input that is
```

Comment:
The `ChangeLog` entry should be a file with the `.txt` extension in the `ChangeLog.d` directory, as otherwise there are conflicts on just about every PR.

See `ChangeLog.d/00README.md`